yarn-project/aztec/src/cli/aztec_start_options.ts

```diff
@@ -108,6 +108,12 @@ export const aztecStartOptions: { [key: string]: AztecStartOption[] } = {
       envVar: 'ETHEREUM_HOSTS',
       parseVal: (val: string) => val.split(',').map(url => url.trim()),
     },
+    {
+      flag: '--l1-consensus-host-url <value>',
```

Comment:
regarding this change & #12787, it makes sense for this to be part of node settings since it's used by multiple components. You should also add `l1ConsensusHostUrl` to `universalOptions`